README.md

```diff
@@ -1,16 +1,19 @@
-![Build Status](https://travis-ci.org/BrunoNicholas/iReporter.svg?branch=Develope) [![Coverage Status](https://coveralls.io/repos/github/BrunoNicholas/iReporter/badge.svg?branch=Develope)](https://coveralls.io/github/BrunoNicholas/iReporter?branch=Develope)
+[![Build Status](https://travis-ci.org/BrunoNicholas/iReporter.svg?branch=Develope)](https://travis-ci.org/BrunoNicholas/iReporter)
+
+(https://coveralls.io/repos/github/BrunoNicholas/iReporter/badge.svg?branch=Develope)](https://coveralls.io/github/BrunoNicholas/iReporter?branch=Develope)
```

Comment:
Correct the syntax for the coverage badge.it should be `[![Coverage Status](https://coveralls.io/repos/github/BrunoNicholas/iReporter/badge.svg?branch=bugfix)](https://coveralls.io/github/BrunoNicholas/iReporter?branch=bugfix)`

Reply:
The correct way is to go to coveralls and click on embed and copy the markdown

Reply:
Thanks so much, @kalsmic , I was not minding much about the markdown format but I have read about it and it makes much sense